css/style.css

```
@@ -131,9 +158,14 @@ img {
 }
 
 .header-logo {
+  grid-area: gap;
   position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
   width: 138px;
   left: 571px;
+  margin: 0 auto;
+  margin-bottom: -22px;
 }
```

```suggestion
.header-logo {
  position: absolute;
  left: 50%;
  transform: translateX(-50%);
}
```

- **grid-area: gap** у этого блока быть не может потому, что он не часть **navigation-list**
- **left: 571px** переопределяет **left: 50%** и центровка работает криво
- ширина не нужна потому как ссылка и так будет равна по ширине и высоте вложенному в нее изображению